src/modules/communication/index.ts

```diff
@@ -0,0 +1,11 @@
+import EmailService from './email-service';
+import SMSService from './sms-service';
+import { SendGridValidationError } from './types';
+import SendGridService from './internals/send-grid-service';
+
+export {
+  EmailService,
+  SMSService,
+  SendGridValidationError,
```

Comment:
Nothing about sendgrid should be visible to outside world from here

Reply:
@jjalan  sir, SendGridValidationError, is just another class initiated by me, it is not from sendgrid. This class is defined in types.ts.

Reply:
Understood, it cannot be in index.ts file for the module.

Sendgrid is an internal detail